MaybeGame/EnemyConfig.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace MaybeGame
+{
+    public static class EnemyConfig
+    {
+        public static Enemy CreateZombie()
+        {
+            var enemy = new Enemy();
+            enemy.StatSet("Zombie");
```

Comment:
Pass enum here

Reply:
That would be easier to modify than passing a string